### ISSUE TITLE
Refactor the Animation / section organizer demo in Flutter Gallery to use LinkedScrollView-like logic

### ISSUE THREAD

**Issue by Hixie** [flutter/flutter#44609]
The Animation / section organizer demo is the last place using the deprecated jumpToWithoutSettling. It's using it as a workaround to the way it was designed. It currently uses scroll notification listeners to force two unrelated lists to share a scroll position. A better solution would be to have them share a scroll position, in a similar way to how the LinkedScrollView test does its thing.

(I'm trying to remove jumpToWithoutSettling, since it's been deprecated since forever.)

**Comment by Hixie** [comment]
cc @HansMuller @Piinks

**Comment by gibbsvjy007** [comment]
I am facing the same issue

**Comment by TahaTesser** [comment]
`jumpToWithoutSettling` is still present in dev/integration_tests/flutter_gallery 
https://github.com/flutter/flutter/blob/e167181444fd8864d1f83be3ca73c599f5a9f508/dev/integration_tests/flutter_gallery/lib/demo/animation/home.dart#L485-L493

Although I can't find in new Flutter Gallery  https://github.com/flutter/gallery

Are you still maintaing/refracitng flutter_gallery in  dev/integration_tests/flutter_gallery?

**Comment by HansMuller** [comment]
Yes it's still part of dev/integration_tests/flutter_gallery because it's still needed for tests and benchmarks. So far, there hasn't been time to update this animation demo.
